x/inflation/genesis.go

```diff
@@ -1,6 +1,7 @@
 package inflation
 
 import (
+	"cosmossdk.io/errors"
```

Comment:
Nit: add `errorsmod` alias to be consistent with codebase
```suggestion
	errorsmod "cosmossdk.io/errors"
```